src/main/webapp/feed.html

```diff
@@ -3,11 +3,83 @@
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>Feed</h1>
+    <div class="container-fluid">
+      <nav class="navbar navbar-expand-sm justify-content-center navbar-background">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+              <img src="images/logo.jpg" class="rounded-circle" alt="logo" height="80" width="100">
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="profile.html">Profile</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="feed.html">Feed</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="matches.html">Matches</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="content">
+        <h1>Feed</h1>
+        <div id="feed" class="carousel slide" data-ride="carousel">
+          <ul class="carousel-indicators">
+            <li data-target="#feed" data-slide-to="0" class="active"></li>
+            <li data-target="#feed" data-slide-to="1"></li>
+            <li data-target="#feed" data-slide-to="2"></li>
+          </ul>
+          <div class="carousel-inner">
+            <div class="carousel-item active">
+              <img src="images/placeholder.jpg" alt="placeholder" width="1100" height="800">
+              <div class="carousel-caption">
+                <h3>NAME</h3>
+                <p>This is an example bio.</p>
+              </div> 
+            </div>
+            <div class="carousel-item">
+              <img src="images/placeholder.jpg" alt="placeholder" width="1100" height="800">
+                <div class="carousel-caption">
+                  <h3>NAME</h3>
+                  <p>This is an example bio.</p>
+                </div> 
+            </div>
+            <div class="carousel-item">
+              <img src="images/placeholder.jpg" alt="placeholder" width="1100" height="800">
+              <div class="carousel-caption">
+                <h3>NAME</h3>
+                <p>This is an example bio.</p>
+              </div> 
+            </div>
+          </div>
+          <a class="carousel-control-prev" href="#feed" data-slide="prev">
+            <span class="carousel-control-prev-icon"></span>
+          </a>
+          <a class="carousel-control-next" href="#feed" data-slide="next">
+            <span class="carousel-control-next-icon"></span>
+          </a>
+        </div> <!--carousel -->
```

Comment:
nit: should this comment be on it's own line?

Reply:
the comment is for the closing div so that anyone looking at the code knows what the div is closing!